### PR TITLE
Added RSVP Logs Tab to All Events

### LINKED ISSUE #829: Create a "RSVP history" on the Track Volunteers page.
Create another tab on events for RSVP History. This will display an RSVP log for that event and it should look something like the Admin log. We want to record ALL RSVP actions: 

- When an RSVP occurs
- When someone un RSVPs
- When someone is deleted from the RSVP table
- When someone is put on the waitlist
- When someone is delete from the waitlist table
- When someone is moved from the waitlist table to the RSVP table an who moved them
- When someone is added to the RSVP table through the "Add Volunteer" modal and who added them

Checkout `rsvpLog` branch to work on this issue. Talk with Anderson when starting.

### PR DESCRIPTION
**Pull request by hoerstl**
We added another tab to all event pages. It displays the history of all RSVPs and waitlist actions that have been taken by admins or by the students themselves. 

Resolves #829

### CODE REVIEW

File: app/templates/events/rsvpLog.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set title = "Rsvp Logs" %}
+{% set page_title = eventData.name %}
+{% set eventPast = event.isPast %}
+{% set tabName = 'rsvpLog' %}
+{% extends "eventNav.html" %}
+
+{% block scripts %}
+  {{super()}}
+  {#Todo: Make new js file#}
```
Comment (AndersonStettner) [inline app/templates/events/rsvpLog.html]: remove Todo if it is done. Do it if it is not done.

**Review by AndersonStettner — CHANGES_REQUESTED** [review]
Resolve merge conflict

File: app/controllers/admin/routes.py [hunk 2]
```diff
@@ -136,6 +137,23 @@ def createEvent(templateid, programid=None):
...
 
+
+@admin_bp.route('/event/<eventId>/rsvp', methods=['GET'])
+def rsvpLogDisplay(eventId):
+    event = Event.get_by_id(eventId)
+    eventData = model_to_dict(event, recurse=False)
+    eventData['program'] = event.singleProgram
+    isProgramManager = g.current_user.isProgramManagerFor(eventData['program'])
+    if g.current_user.isCeltsAdmin or (g.current_user.isCeltsStudentStaff and isProgramManager):
+        allLogs = EventRsvpLogs.select(EventRsvpLogs, User).join(User).order_by(EventRsvpLogs.createdOn.desc()).where(EventRsvpLogs.event_id == eventId)
```
Comment (AndersonStettner) [inline app/controllers/admin/routes.py]: We should move the `.order_by` to the end. To paraphrase Brian, "Always put your `.order_by` and `.group_by` at the end of the query.

**Review by AndersonStettner — CHANGES_REQUESTED** [review]

File: app/controllers/main/routes.py [hunk 3]
```diff
@@ -200,7 +201,7 @@ def ban(program_id, username):
         banUser(program_id, username, banNote, banEndDate, g.current_user)
         programInfo = Program.get(int(program_id))
         flash("Successfully banned the volunteer", "success")
-        createLog(f'Banned {username} from {programInfo.programName} until {banEndDate}.')
+        createAdminLog(f'Banned {username} from {programInfo.programName} until {banEndDate}.')
```
Comment (BrianRamsay) [inline app/controllers/main/routes.py:204]: You never import `createAdminLog`

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

File: app/models/eventRsvpLogs.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+from app.models import* 
+from app.models.user import User 
+from app.models.event import Event
+
+class EventRsvpLogs(baseModel):
```
Comment (BrianRamsay) [inline app/models/eventRsvpLogs.py]: Class and table names should be singular, as each record is a single log

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

**Review by BrianRamsay — APPROVED** [review]

**Pull request merged**